mail_tracking/models/mail_tracking_email.py

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+import urlparse
+import time
+from datetime import datetime
+
+from openerp import models, api, fields, tools
+import openerp.addons.decimal_precision as dp
+
+_logger = logging.getLogger(__name__)
+
+
+class MailTrackingEmail(models.Model):
+    _name = "mail.tracking.email"
+    _order = 'time desc'
+    _rec_name = 'name'
+    _description = 'MailTracking email'
+
+    name = fields.Char(string="Subject", readonly=True)
+    timestamp = fields.Float(
+        string='UTC timestamp', readonly=True,
+        digits=dp.get_precision('MailTracking Timestamp'))
+    time = fields.Datetime(string="Time", readonly=True)
+    date = fields.Date(string="Date", readonly=True)
+    mail_message_id = fields.Many2one(
+        string="Message", comodel_name='mail.message', readonly=True)
+    mail_id = fields.Many2one(
+        string="Email", comodel_name='mail.mail', readonly=True)
+    partner_id = fields.Many2one(
+        string="Partner", comodel_name='res.partner', readonly=True)
+    recipient = fields.Char(string='Recipient email', readonly=True)
+    sender = fields.Char(string='Sender email', readonly=True)
+    state = fields.Selection([
+        ('error', 'Error'),
+        ('deferred', 'Deferred'),
+        ('sent', 'Sent'),
+        ('delivered', 'Delivered'),
+        ('opened', 'Opened'),
+        ('rejected', 'Rejected'),
+        ('spam', 'Spam'),
+        ('unsub', 'Unsubscribed'),
+        ('bounced', 'Bounced'),
+        ('soft-bounced', 'Soft bounced'),
+    ], string='State', index=True, readonly=True, default=False,
+        help=" * The 'Error' status indicates that there was an error "
+             "when try to sent the email, for example, 'No valid recipient'\n"
+             " * The 'Sent' status indicates that message was succesfully "
+             "sent via outgoing email server (SMTP).\n"
+             " * The 'Delivered' status indicates that message was "
+             "succesfully delivered to recipient Mail Exchange (MX) server.\n"
+             " * The 'Opened' status indicates that message was opened or "
```

Comment:
In this scenario "Opened" is correct, as we are referring to the past-tense form of it - the message was Opened. Comparing this to account.invoice Open status, which is indicating that the record is currently Open